packages/rancher-logging/generated-changes/overlay/templates/_generic_logging.yaml

```diff
@@ -65,11 +65,9 @@ spec:
     configReloaderImage:
       repository: {{ template "system_default_registry" . }}{{ .Values.images.config_reloader.repository }}
       tag: {{ .Values.images.config_reloader.tag }}
-    {{- if not .Values.disablePvc }}
```

Comment:
@superseb Upgrading fluentd won't have any effect on this issue.  The reason `fluentd.bufferStorageVolume` was not getting configured from the chart values was because of this check. So I used the images which upstream logging-operator uses. https://github.com/banzaicloud/logging-operator/blob/3.17.4/pkg/sdk/logging/api/v1beta1/logging_types.go#L115